info: CustomCategory_TestSignal[0]
      PreHook:TestSignal,test-message-id,test-trace-id
fail: CustomCategory_TestSignal[0]
      ExceptionHook:TestException,test-message-id,test-trace-id
      Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals+TestException: test exception
         at Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalHandler.Handle(TestSignal signal, CancellationToken cancellationToken) in {ProjectDirectory}Signalling/LoggingMiddlewareTestSignals.cs:line FIXED
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Signalling/LoggingSignalMiddleware.cs:line FIXED
         at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx)
         at Conqueror.Signalling.SignalPipelineRunner`1.<>c__DisplayClass3_2.<Execute>b__1(TSignal signal, CancellationToken token) in {SolutionDirectory}src/Conqueror/Signalling/SignalPipelineRunner.cs:line FIXED

